backends/qualcomm/tests/models.py

```diff
@@ -1079,3 +1079,19 @@ def forward(self, x, y):
         x = x.view(new_shape)
         x = x.permute(0, 2, 1, 3)
         return torch.matmul(x, y.transpose(-1, -2))
+
+class draw_graph_model(torch.nn.Module):
```

Comment:
Camel form on class name would be better, please also put this in alphabetical order.